interfaces/api/v1/prompts/marketplace.py

```
@@ -0,0 +1,130 @@
+# 覆盖管理
+# ------------------------------------------------------------------
+
+
+@router.post("/override", summary="激活节点覆盖")
+def activate_override(payload: OverrideRequest) -> Dict[str, str]:
+    try:
+        registry = _get_registry()
+        registry.activate_override(payload.node_name, payload.pack_id)
+        return {"status": "ok", "node_name": payload.node_name, "pack_id": payload.pack_id}
+    except KeyError as e:
+        raise HTTPException(status_code=404, detail=str(e))
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)